rpa-recorder/6.6/modules/ROOT/pages/user-interface.adoc

```diff
@@ -185,10 +185,13 @@ Select this option to generate a XES process file after the recording session st
 * *Generate debug file (contains recorded events, window handles and system data)* 
 +
 Select this option to generate a debug file after the recording session stops.
-* *Screenshot Settings* 
-** *Image quality*: 
+* *Enable element highlighting during Bot Task recordings.*
```

Comment:
This feature had not been added when it was released previously, so I'm adding it with this release.